main/githooks.py

```diff
@@ -19,7 +17,7 @@
 # Absolute file size limit (in MB) - it's 100MB on github.com
 HARD_SIZE_THRESHOLD = 99.0
 # Internal file size limit (in MB) - allow if commit message includes marker
-SOFT_SIZE_THRESHOLD = 10.0
```

Comment:
I wondered why the size threshold was being raised from 10MB?  To me it feels reasonable for a warning.

Reply:
How much lower than the github hard limit should our internal LARGE FILE limit be?

Reply:
(I should update the branch with the v1 code)

Reply:
I will set it to 5MB as decided this morning.